app/controllers/answer_votes_controller.rb

```diff
@@ -1,61 +1,26 @@
 class AnswerVotesController < ApplicationController
-  before_action :set_answer_vote, only: [:show, :edit, :update, :destroy]
+  before_filter :authenticate_user!, only: [:create]
```

Comment:
If you use Rails 4+, then it is better to use `before_action` instead on `before_filter`